tools/lib/provision.py

```diff
@@ -322,6 +328,13 @@ def main(options):
         from zerver.lib.str_utils import force_bytes
         from zerver.lib.test_fixtures import is_template_database_current
 
+        # change settings for docker compose
+        from django.conf import settings
+        if options.is_docker_compose:
```

Comment:
I don't think this is needed, a modification to the `configure-rabbitmq` script should look better instead of an extra case for it, as seen in my docker-zulip repository: https://github.com/galexrt/docker-zulip/blob/master/custom_zulip_files/scripts/setup/configure-rabbitmq.

Reply:
Added to the first commit.

Reply:
@galexrt maybe I am interpreting your suggestion wrongly: I made it so that `scripts/setup/configure-rabbitmq` always uses the `-n` flag for `rabbitmqctl`. I added this, and it breaks Travis currently. Or do you mean to add the flag to `scripts/setup/configure-rabbitmq`, e.g. `--docker-compose` so that it adds the `RABBITMQ_NODE` env var via `$("$(dirname "$0")/../get-django-setting" RABBITMQ_HOST)`?

Reply:
@rht I think maybe a better approach then always use `-n` flag would be to have a by default empty flag named `RABBITMQ_FLAGS` and set it with the `-n` flag in cases like this when it is needed maybe this works better for Travis then.

Reply:
I experimented with always having the `-n` flag, it broke Travis because it somehow couldn't reach to `localhost`. The current setup of only setting `RABBITMQ_NODE` for docker-compose works for now. i.e. there is not a general solution that captures both the external rabbitmq host in another container and a local rabbitmq within one container.